browser/src/libs/github/file_info.test.ts

```diff
@@ -0,0 +1,18 @@
+import { readFile } from 'mz/fs'
+import { getFilePath } from './util'
+
+const tests = [
+    ['github.com/blob/vanilla/page.html', 'shared/src/api/extension/types/url.ts'],
+    ['github.com/blob/refined-github/page.html', 'shared/src/api/extension/types/url.ts'],
+    ['ghe-2.14.11/blob/vanilla/page.html', 'bench_test.go'],
+    ['ghe-2.14.11/blob/refined-github/page.html', 'bench_test.go'],
+]
+
+describe('github/file_info', () => {
+    for (const [fixture, expectedFilePath] of tests) {
+        it(`getFilePath() finds the file path in ${fixture}`, async () => {
```

Comment:
```suggestion
        it(`finds the file path in ${fixture}`, async () => {
```

This should read like an English sentence.